charts/opensearch/templates/statefulset.yaml

```diff
@@ -366,8 +369,11 @@ spec:
           subPath: tenants.yml
         {{- end }}
         {{- if .Values.securityConfig.config.data  }}
-        - mountPath: {{ .Values.securityConfig.path }}
-          name: security-config-complete
+        {{- range $key, $_ := .Values.securityConfig.config.data  }}
+        - mountPath: {{ $.Values.securityConfig.path }}/{{ $key }}
+          name: security-config
```

Comment:
Doesn't this need to be `name: security-config-{{ $key }} or some kind of unique identifier? Otherwise this breaks since you are using subPath and the secret can have multiple keys.

Reply:
You can use the same volume for mounting separate files (and not entirely volume content) by using subPath. And you can do it multiple times.

Reply:
@bkomraz1 You can. Stupid me. Thank you.